wbooks/src/config/reactotronConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { NativeModules } from 'react-native';
+import Reactotron, { trackGlobalErrors, overlay } from 'reactotron-react-native';
+
+interface Tron {
+  log: (...args: any[]) => void;
+  clear: () => void;
+  customCommand: (arg: any) => void;
+  display: (arg: any) => void;
+}
+declare global {
+  interface Console {
+    tron: Tron;
+  }
+}
+
+if (__DEV__) {
+  const { scriptURL } = NativeModules.SourceCode;
+  const scriptHostname = scriptURL.split('://')[1].split(':')[0];
+  Reactotron.configure({ name: 'appName', host: scriptHostname })
```

Comment:
Wrong! `'appName'` is not the real app name, [check](https://github.com/wolox-training/av-react-native/blob/master/wbooks/app.json) again